packages/web-react/docs/stories/feature-flags/Overview.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Experimental/Feature Flags" />
+
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The `@lmc-eu/spirit-web-react` codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in JavaScript, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
+how to configure it. Folders and stories within this section in Storybook show
+components with all feature flags turned on.
+
+## Current feature flags
+
+Flags prefixed with `enable-*` contain features we'd like consuming projects to
+test and give us feedback on. They're generally stable and unlikely to change
+but may change based on your feedback.
+
+Flags prefixed with `enable-v3-*` are stable and won't change. They will be
+marked as `true` or "on" by default in the next major version, v3.
+
+For more details on this approach, see the
+[feature flag documentation](https://github.com/lmc-eu/spirit-design-system/blob/main/docs/contributting/experimental-code.md)
+in the Carbon monorepo.
+
+| Flag | Description | Default | SASS flag |
+| ---- | ----------- | ------- | --------- |
+
+## Turning on feature flags in SASS
+
+In SASS, you can enable feature flags in any of your stylesheets. Most often
+this is done at the root/entrypoint stylesheet.
+
+```sass
+@use '~@lmc-eu/spirit-web/scss/settings/feature-flags' with (
+  $enable-modal-fullscreen: true
+);
+
+@use '~@lmc-eu/spirit-web/scss/foundation';
+```
+
+Feature flags can also be enabled via the provided `enable()` mixin
+
+```sass
+@use '@lmc-eu/spirit-web/scss/feature-flags';
+@use '@lmc-eu/spirit-web';
+
+@include feature-flags.enable('enable-experimental-tile-contrast');
```

Comment:
we don't have this infrastructure now. Currently, we support only the former option, but we also allow classname featureflags to be enabled locally.

Reply:
Ok, I have removed the `enable`.